ansible/roles/linux-ansible-init/tasks/main.yml

```diff
@@ -33,3 +35,4 @@
     daemon_reload: yes
     name: ansible-init.service
     enabled: yes
+  ignore_errors: yes
```

Comment:
Why are we ignoring errors here? That seems wrong.

Reply:
Mainly so that the install continues even if the systemd service installation fails. I guess it is more useful for debugging the image build, so it could be removed

Reply:
We definitely shouldn't continue if the installation of the systemd service fails right? That feels like something I would want to know during the build process...

Reply:
Yes, and no! During my investigations of this problem, I found that there is a paucity of logging when this step failed, even with all the "vees" (`-vvvv`). To determine the problem, I needed the build to complete so I could fire up a VM and investigate with `systemctl` and other tools. 

So on one hand, with `ignore_errors` the problem is notified but it allows investigation. On the other, it probably won't be a problem in the future again?

Happy to do what you like, just thought the background might be useful

Reply:
@jjo93sa 

When debugging issues like that, you should run `packer` manually with the `-debug` flag. That allows you to SSH into the machine that will eventually become the image _while it is being built_ to investigate. Stuff like this shouldn't end up in the final PR IMHO.

Reply:
I should write up some docs on how to do that...